client/verta/verta/registry/entities/_model.py

```diff
@@ -1090,7 +1090,12 @@ def _create_proto_internal(
 
     def set_pii(self, pii):
         """
-        Note that you *MUST* update any live endpoints running this model in order to propagate this change.
+        Updates the PII value of this Registered Model. True indicates that the model ingests PII.
```

Comment:
In case someone reading this doesn't know what PII is, and isn't sure if this is the method for them.

```suggestion
        Updates the PII value of this Registered Model.
        
        ``True`` indicates that the model ingests personally identifiable information.
```